okhttp-tests/src/test/java/okhttp3/ProtocolTest.java

```diff
@@ -0,0 +1,34 @@
+package okhttp3;
+
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+
+public class ProtocolTest {
+    @Test
+    public void testGetKnown() throws IOException {
+        assertEquals(Protocol.HTTP_1_0, Protocol.get("http/1.0"));
+        assertEquals(Protocol.HTTP_1_1, Protocol.get("http/1.1"));
+        assertEquals(Protocol.SPDY_3, Protocol.get("spdy/3.1"));
+        assertEquals(Protocol.HTTP_2, Protocol.get("h2"));
+        assertEquals(Protocol.H2C, Protocol.get("h2c"));
+        assertEquals(Protocol.QUIC, Protocol.get("quic"));
+    }
+
+    @Test(expected = IOException.class)
```

Comment:
nit: I expect (no pun intended) Jesse will ask you to turn this into 

```
try {
  X; 
  fail();
} catch (IOException expected) { 
assertEquals("xxxx", expected.getMessage()); 
}
```

Reply:
Yes please.